<ui version="4.0" >
 <class>GamesDialog</class>
 <widget class="QWidget" name="GamesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>6</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="verticalSpacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="images.qrc" >:/images/package_games1.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt; font-weight:600;">Games Launcher&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>431</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3" >
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label" >
           <property name="font" >
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text" >
            <string>Game:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="RSComboBox" name="gameComboBox" >
           <item>
            <property name="text" >
             <string>GameType: 0. Want to Add your Game here?</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>GameType: 1. Get In Touch with the developers</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>GameType: 2.</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label_4" >
           <property name="font" >
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text" >
            <string>Title / Comment</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="gameNameEdit" />
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="createButton" >
           <property name="font" >
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text" >
            <string>Create New Game</string>
           </property>
           <property name="icon" >
            <iconset resource="images.qrc" >
             <normaloff>:/images/kbackgammon.png</normaloff>:/images/kbackgammon.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkInviteAll" >
           <property name="font" >
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text" >
            <string>Invite All Friends</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0" >
      <widget class="QSplitter" name="splitter" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTreeWidget" name="gameTreeWidget" >
        <property name="contextMenuPolicy" >
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <column>
         <property name="text" >
          <string>Game Type</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Server</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Status</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Comment</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>GameID</string>
         </property>
        </column>
       </widget>
       <widget class="QTreeWidget" name="peertreeWidget" >
        <property name="contextMenuPolicy" >
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <column>
         <property name="text" >
          <string>Player</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Invite</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Interested</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Accept</string>
         </property>
        </column>
       </widget>
      </widget>
     </item>
     <item row="2" column="0" >
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QPushButton" name="deleteButton" >
         <property name="font" >
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Delete</string>
         </property>
         <property name="icon" >
          <iconset resource="images.qrc" >
           <normaloff>:/images/reset.png</normaloff>:/images/reset.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="inviteButton" >
         <property name="font" >
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Invite</string>
         </property>
         <property name="icon" >
          <iconset resource="images.qrc" >
           <normaloff>:/images/dadd.png</normaloff>:/images/add_user24.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="moveUpButton" >
         <property name="font" >
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Move Player</string>
         </property>
         <property name="icon" >
          <iconset resource="images.qrc" >
           <normaloff>:/images/up.png</normaloff>:/images/up.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="moveDownButton" >
         <property name="font" >
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Move Player</string>
         </property>
         <property name="icon" >
          <iconset resource="images.qrc" >
           <normaloff>:/images/down_24x24.png</normaloff>:/images/down_24x24.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="playButton" >
         <property name="font" >
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Play Game</string>
         </property>
         <property name="icon" >
          <iconset resource="images.qrc" >
           <normaloff>:/images/startall.png</normaloff>:/images/startall.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RSComboBox</class>
   <extends>QComboBox</extends>
   <header>gui/common/RSComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images.qrc" />
 </resources>
 <connections/>
</ui>
